include/dxc/DXIL/DxilInstructions.h

```diff
@@ -12,11 +12,11 @@
 
 #pragma once
 
+#include "dxc/DXIL/DxilOperations.h"
 #include "llvm/IR/Constants.h"
 #include "llvm/IR/Instruction.h"
 #include "llvm/IR/Instructions.h"
 
-// TODO: add correct include directives
```

Comment:
I'm not sure this means what you think it means. Regardless, I think it is out of date.